code/modules/mob/living/silicon/robot/laws.dm

```diff
@@ -28,6 +29,7 @@
 
 	who << "<b>Obey these laws:</b>"
 	laws.show_laws(who)
+	// TODO: Update to new antagonist system.

```

Comment:
That's good to know, thanks. I'll hold off on porting it until those issues are resolved.